styles.css

```diff
@@ -0,0 +1,197 @@
+nav {
+  position: relative;
+  z-index: 99 !important;
```

Comment:
- [ ] please avoid using `!important`, if it doesn't work without it, it's probably because you're linking your style.css file **before** the bootstrap one, so bootstrap overwrites it. 
remember, CSS shows the last property that has been called

[Here's a quick example I made](https://codepen.io/SpaYco/pen/gOwNqMm), notice how the block is red, even though I said blue at first, that's because red was the last `background-color` i called on the `div`, same happens when you link different stylesheets, it can also happen sometimes with the class names order. hope this helped.